src/shipit_code_coverage/default.nix

```diff
@@ -63,6 +63,7 @@ let
       hook = mkTaskclusterHook {
         name = "Shipit task aggregating code coverage data";
         owner = "mcastelluccio@mozilla.com";
+        schedule = [ "0 0 0 * * 0" ]; # every week
```

Comment:
I checked this one as the nodejs cron format is tricky, this is indeed once per week. :+1: